web/src/components/CityContentFooter.tsx

```diff
@@ -31,48 +22,21 @@ type CityContentFooterProps = {
 }
 
 const CityContentFooter = ({ city, language, mode = 'normal' }: CityContentFooterProps): ReactElement => {
-  const { viewportSmall } = useWindowDimensions()
-  const { aboutUrls, privacyUrls, accessibilityUrls } = buildConfig()
-  const { t } = useTranslation(['layout', 'settings'])
-  const aboutUrl = aboutUrls[language] || aboutUrls.default
-  const privacyUrl = privacyUrls[language] || privacyUrls.default
-  const accessibilityUrl = accessibilityUrls?.[language] ?? accessibilityUrls?.default
-  const disclaimerPath = pathnameFromRouteInformation({
-    route: DISCLAIMER_ROUTE,
-    cityCode: city,
-    languageCode: language,
-  })
-  const licensesPath = pathnameFromRouteInformation({
-    route: LICENSES_ROUTE,
-  })
-
-  const linkItems = [
-    { to: disclaimerPath, text: t('imprint') },
-    { to: aboutUrl, text: t('settings:about', { appName: buildConfig().appName }) },
-    { to: privacyUrl, text: t('privacy') },
-    { to: licensesPath, text: t('settings:openSourceLicenses') },
-    ...(accessibilityUrl ? [{ to: accessibilityUrl, text: t('accessibility') }] : []),
-  ]
-  const Links = (
-    <>
-      {linkItems.map((item, index) => (
-        <React.Fragment key={item.to}>
-          <Link to={item.to}>{item.text}</Link>
-          {viewportSmall && index < linkItems.length - 1 && <Divider />}
-        </React.Fragment>
+  const linkItems = getFooterLinks({ city, language })
+  const FooterContent = (
+    <StyledList
+      items={linkItems.map(item => (
+        <FooterLink key={item.to} to={item.to} text={item.text} mode={mode} />
       ))}
-    </>
+      horizontal={mode !== 'sidebar'}
+    />
   )
 
   if (mode === 'sidebar') {
-    return (
-      <Footer>
-        <SidebarFooterContainer>{Links}</SidebarFooterContainer>
-      </Footer>
-    )
+    return <Footer>{FooterContent}</Footer>
   }
 
-  return <Footer overlay={mode === 'overlay'}>{Links}</Footer>
+  return mode === 'overlay' ? <OverlayFooter>{FooterContent}</OverlayFooter> : <Footer>{FooterContent}</Footer>
```

Comment:
🔧 This can be simplified, for modes `sidebar` and `normal` exactly the same thing is rendered. The check for `mode === 'sidebar'` can therefore be removed here (above its still necessary though).
``` tsx
  return mode === 'overlay' ? <OverlayFooter>{FooterContent}</OverlayFooter> : <Footer>{FooterContent}</Footer>
```